src/components/Map.js

```diff
@@ -32,11 +35,15 @@ export default function Map() {
     setShowAll(!showAll);
   };
 
-  const handleCurrentPositions = () => {
-    setPosition(getLocation());
-  };
-
   const handleUserFavorites = async () => {
+    if (user.favoriteRegions.length > 2) {
+      setMessage('이미 3개의 관심지역이 있습니다.');
+      return;
+    } else if (user.favoriteRegions.includes(currentAddress)) {
+      setMessage('이미 이 지역이 관심지역에 있습니다.');
+      return;
+    }
+
```

Comment:
```suggestion
    if (user.favoriteRegions.includes(enteredRegion)) {
      setMessage('이미 선택된 관심지역입니다.');
      return;
    }

    if (user.favoriteRegions.length === 3) {
      setMessage('관심지역의 갯수를 초과하였습니다.');
      return;
    }

```
관심매물 알람 메시지 문구와 비슷한 맥락으로 메시지를 작성하면 좋을 것 같아요

Reply:
조금 더 사용자 친화적인 메세지네요 !!
좋은것 같습니다